tdrs-backend/tdpservice/parsers/schema_defs/ssp/m3.py

```diff
@@ -13,50 +13,146 @@
     preparsing_validators=[
         validators.notEmpty(start=19, end=60),
     ],
-    postparsing_validators=[],
+    postparsing_validators=[
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.matches(1),
+                  result_field='SSN', result_function=validators.validateSSN(),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_HISPANIC', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_AMER_INDIAN', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_ASIAN', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_BLACK', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_HAWAIIAN', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RACE_WHITE', result_function=validators.isInLimits(1, 2),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='RELATIONSHIP_HOH', result_function=validators.isInLimits(4, 9),
+            ),
+        validators.if_then_validator(
+                  condition_field='FAMILY_AFFILIATION', condition_function=validators.oneOf((1, 2)),
+                  result_field='PARENT_MINOR_CHILD', result_function=validators.oneOf((1, 2, 3)),
+                  # incl 1 - not allowed in c2
+            ),
+        validators.if_then_validator(
```

Comment:
@jtimpe not seeing this one:

![ssps1p2](https://github.com/raft-tech/TANF-app/assets/63075587/4478c395-9de4-43e5-a358-6384e91b0a70)